client/app/pages/data-sources/CreateDataSource.jsx

```diff
@@ -3,26 +3,61 @@ import Steps from 'antd/lib/steps';
 import { find } from 'lodash';
 import { react2angular } from 'react2angular';
 import { DataSource } from '@/services/data-source';
+import navigateTo from '@/services/navigateTo';
 import TypePicker from '@/components/TypePicker';
 import { DynamicForm } from '@/components/dynamic-form/DynamicForm';
 import helper from '@/components/dynamic-form/dynamicFormHelper';
 
 const { Step } = Steps;
 
+const StepEnum = {
+  SELECT_TYPE: 0,
+  CONFIGURE_IT: 1,
+  DONE: 2,
+};
+
+const HELP_LINKS = {
+  athena: 'https://redash.io/help/data-sources/amazon-athena-setup',
+  bigquery: 'https://redash.io/help/data-sources/bigquery-setup',
+  url: 'https://redash.io/help/data-sources/querying-urls',
+  mongodb: 'https://redash.io/help/data-sources/mongodb-setup',
+  google_spreadsheets: 'https://redash.io/help/data-sources/querying-a-google-spreadsheet',
+  google_analytics: 'https://redash.io/help/data-sources/google-analytics-setup',
+  axibasetsd: 'https://redash.io/help/data-sources/axibase-time-series-database',
+  results: 'https://redash.io/help/user-guide/querying/query-results-data-source',
+};
+
 class CreateDataSource extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { dataSourceTypes: [], selectedType: null };
+    this.state = { dataSourceTypes: [], selectedType: null, currentStep: StepEnum.SELECT_TYPE };
     DataSource.types(dataSourceTypes => this.setState({ dataSourceTypes }));
+    this.topRef = React.createRef();
   }
 
+  scrollToTop = () => {
+    window.scrollTo({ top: this.topRef.current.offsetTop, behavior: 'smooth' });
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo), this way to use `window.scrollTo` not supported in IE/Safari. Can you check it?

Reply:
I was aware that sth like this could show up :thinking:. Thanks, I'll check this.